lessons/6-data-manipulation/slides.qmd

```diff
@@ -69,7 +96,7 @@ We might need to create a repo for R scripting for this workshop.
 
 -   ‼️ Extra installs over "base R"
 -   ‼️ Very different syntax than "base R"
--   ‼️ Updates rapidly
+-   ‼️ Updates rapidly (can be a reproducibility challenge)
```

Comment:
You might note (on the slides or just verbally) that the tidyverse _is_ pretty exceptional when it comes to having helpful warning and error messages that help you update your code when you need to.  Actually _breaking_ changes are pretty rare, and when they do happen the error messages usually give you instructions on how to update your code.